recipes/eiger-io/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "eiger-io" %}
+{% set version = "2.0.4" %}
+{% set sha256 = "40f79e360c300a71dd358f5881c3e361ce08a0ce2780e953e0531bef27c65ad6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NSLS-II/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - h5py
+    - hdf5-external-filter-plugins-bitshuffle
+    - hdf5-external-filter-plugins-lz4
+    - numpy
+    - pims
+    - six
+
+test:
+  imports:
+    - eiger_io
```

Comment:
`eiger_io.__init__` is pretty much empty. Can you add an import that better tests whether the package will work (and has all its necessary dependencies)?

Reply:
Suggested imports:
```yaml
- eiger_io.fs_handler
- eiger_io.fs_handler_dask
- eiger_io.tools
- eiger_io._version
```